product-service/src/functions/post-products/handler.ts

```diff
@@ -0,0 +1,85 @@
+import 'source-map-support/register';
+import {formatJSONResponse, ValidatedEventAPIGatewayProxyEvent} from '@libs/apiGateway';
+import {middyfy} from '@libs/lambda';
+import {Pool} from 'pg';
+import schema from '@functions/post-products/schema';
+
+const {PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD} = process.env;
+const dbOptions = {
+    host: PG_HOST,
+    port: PG_PORT,
+    database: PG_DATABASE,
+    user: PG_USERNAME,
+    password: PG_PASSWORD,
+    ssl: {
+        rejectUnauthorized: false
+    },
+    connectionTimeoutMillis: 1000
+};
```

Comment:
It's not a good practice to create a DB connection object in each lambda.
I would like to advise you to create a separate data access layer of your service with encapsulated logic of all DB connections/querying/disconnection.